ocs_ci/utility/metadata_utils.py

```diff
@@ -0,0 +1,258 @@
+"""
+Module that contains all operations related to add metadata feature in a cluster
+
+"""
+
+import logging
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+
+log = logging.getLogger(__name__)
+
+
+def check_setmetadata_availability(pod_obj):
+    """
+    Check setmetadata parameter is available or not for cephfs and rbd plugin pods
+
+    """
+    plugin_provisioner_pod_objs = pod.get_all_pods(
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+        selector=["csi-cephfsplugin-provisioner", "csi-rbdplugin-provisioner"],
+    )
+    log.info(f"list of provisioner pods---- {plugin_provisioner_pod_objs}")
+    response = pod.validate_pods_are_respinned_and_running_state(
+        plugin_provisioner_pod_objs
+    )
+    log.info(response)
+    for plugin_provisioner_pod in plugin_provisioner_pod_objs:
```

Comment:
This will return once the first pod has "--setmetadata=true".
Shouldn't we check once per RBD provisioner pod and once per CephFS?

Reply:
Addressed in latest commit.